README.md

```diff
@@ -522,6 +522,30 @@ const tag = defineCE(
 );
 ```
 
+### Components with Motion
+
+Components with animations or transitions can cause inconsistent vdiff snapshots, since small timing variations to the snapshot can catch the motion at slightly different points along its path. To help address this, `d2l-test-runner` configures the browser with the `prefers-reduced-motion` setting enabled.
+
+To opt-in to this setting, adjust the component's animation/transition CSS such that it only applies the motion when `prefers-reduced-motion` is set to `no-preference`.
+
+For example:
+
+```css
+my-elem {
+  opacity: 1;
+}
+my-elem.faded-out {
+  opacity: 0;
+}
+@media (prefers-reduced-motion: no-preference) { 
```

Comment:
Interesting, we always see to do the opposite in `core` (turn _off_ the transition with the media query, rather than turn it on).  Are we planning to switch?

Reply:
New stuff in core (e.g. collapsible panel) should be doing it this way. The reason older stuff doesn't is for backwards compatibility with browsers that didn't yet support the setting. We're well past caring about them though.